scripts/zones/Port_Windurst/Zone.lua

```diff
@@ -15,6 +15,11 @@ end;
 
 function onZoneIn(player,prevZone)
     local cs = -1;
+
+    if ENABLE_ROV and player:getCurrentMission(ROV) == dsp.mission.id.rov.RHAPSODIES_OF_VANADIEL and player:getMainLvl()>=3 then
```

Comment:
You have to explicitly check for 1 here, only `nil` and `false` are truly 'false' in Lua.

Also if/until the quest rewrite stuff makes it in to master, it still makes sense to have this after the opening cutscene, nothing trumps that.

(Both these issues exist with the starting cutscenes you added for Seekers of Adoulin as well)